config/config.go

```diff
@@ -258,6 +259,7 @@ var defaultConf = Config{
 		FeedbackProbability: 0,
 		QueryFeedbackLimit:  1024,
 		PseudoEstimateRatio: 0.7,
+		AutoAnalyzeRatio:    1e100,
```

Comment:
This number is still unsafe, I think we should use another special one to explicitly specify that auto analyze is turned off.

Reply:
why not just set it to -1 ?

Reply:
No, it is safe.
-1 make the logic complex.